static/js/protein/page.tsx

```diff
@@ -53,13 +53,26 @@ interface PageStateType {
   data: GraphNodes;
 }
 
-// stores the variant id, tissue name, log fold change value, and log fold change confidence interval
+/* stores the association id, variant id, tissue name, log fold change value, and log fold change confidence interval
```

Comment:
I meant add the comment above the field, so it would look like

```
export interface ProteinVarType {
// genetic variant gene association dcid
associationID: string;
// reference id of the variant
id: string;
...
```

Reply:
Done!